app/src/main/java/com/github/se/signify/model/user/User.kt

```diff
@@ -8,5 +8,8 @@ data class User(
     val friendRequests: List<String> = emptyList(),
     val friends: List<String> = emptyList(),
     val ongoingChallenges: List<String> = emptyList(), // New field for ongoing challenges
-    val pastChallenges: List<String> = emptyList() // New field for past challenges
+    val pastChallenges: List<String> = emptyList(), // New field for past challenges
+    val lastLoginDate: String = "",
+    val currentStreak: Long,
+    val highestStreak: Long
```

Comment:
We can give default value to keep coherence with other element defined in the class.

Reply:
Thanks for pointing this out. It's done.